HW_1/task_2.py

```diff
@@ -0,0 +1,4 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+n = int(input("Введите трехзначное число: "))
+print(n // 100 + (n // 10) % 10 + n % 10)
```

Comment:
Верно, хотя скобки не нужны
```suggestion
print(n // 100 + n // 10 % 10 + n % 10)
```